### DISCUSSION TITLE
Where to get pre-compiled binary for GNU/Linux?

### DISCUSSION THREAD

**Discussion by azzamsa** [rcaelers/workrave#299] [General]
I can't find it in the [snapshot](https://workrave.org/snapshots/) nor in the release artifact.
It only contains windows binary (exe), windows portable, and source code.

Am I missing something?

**Comment by rcaelers** [comment]
We don't provide any pre-compiled binaries for Linux. Most Linux distributions have a Workrave package. See https://workrave.org/download/ (And select Linux/Unix)

**Reply by azzamsa** [reply]
The distribution package is too old. I think it's doable to provide pre-compiled binary on the release page using GitHub action.

I have successfully done this in some of my Rust projects. When the repository is tagged the CI will run the build, and upload the appropriate artifact to the release page.

If this is OK, I will try to build workrave locally then implement the step to Github action. As I never work with C++ before.

**Comment by azzamsa** [comment]
Obsolete as the AppImage already provided. https://github.com/rcaelers/workrave/issues/140#issuecomment-988158687
